### PR TITLE
ci(core): Load config schema after process.env has been overwritten (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**

### CODE REVIEW

File: packages/cli/src/config/index.ts [hunk 1]
```diff
@@ -25,6 +24,8 @@ if (inE2ETests) {
 	dotenv.config();
 }
 
+// Load schema after process.env has been overwritten
+import { schema } from './schema';
```
Comment (tomi) [inline packages/cli/src/config/index.ts:28]: I assume this is the change the PR title mentions? The other changes change from where the static dir is served from. Should that be a separate commit that is also a changelog entry?

Comment (netroy) [reply packages/cli/src/config/index.ts:28]: yeah. this was the most important change. but the removal of `InstanceSettings` from `packages/cli/src/constants.ts` is also important to make sure that `InstanceSettings` are created after `dotenv.config()` has run.

**Review by tomi — DISMISSED** [review]
Nice one 🥇 ~~Only one question: do we need to have a changelog entry for this, since this changes the path used for static cache? (I assume the `no-changelog` in the commit means it will not be included in automatically generated changelog)~~

E: Nevermind, the default stays the same

**Comment by netroy** [comment]
> this changes the path used for static cache?

The path is still the same as before: `~/.cache/n8n/public`

**Review by tomi — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.16.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.16.0)

File: packages/cli/src/config/index.ts (hunk 1 above)
Comment (weiqingtao) [reply packages/cli/src/config/index.ts:28]: > I assume this is the change the PR title mentions? The other changes change from where the static dir is served from. Should that be a separate commit that is also a changelog entry?

hello,How to modify the path to read env?
